docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md

```diff
@@ -4,41 +4,34 @@ sidebar_position: 2
-:::
+**Step 2:** Fully exit your validator from the beacon chain. You may monitor its activity via beaconcha.in/validator/[yourvalidatorid] .
+
+**Step 3:** Wait for the final beacon chain withdrawal to be deposited to your EigenPod. There can be a lag of up to 24 hours to 7 days between the validator appearing as "exited" and the withdrawal amount deposited to EigenPod. Please see the "Withdrawals" tab and "Time" column for your validator via beaconcha.in/validator/[yourvalidatorid]#withdrawals .
+
+**Step 4:**. Queue the full withdrawal from your Eigenpod. Navigate to the **Beacon Ether** asset page in the (EigenLayer App)[https://app.eigenlayer.xyz/].
 
-**Step 1:** Enter the amount in the text box next to the Unstake button.
+**Step 5:** Enter the amount in the text box next to the Unstake button.
 
 You can queue a withdrawal for any amount. However, when you choose to complete the withdrawal, you will be required to exit enough validators and have the proofs generated for the required amount.
 
 ![](/img/googleusercontentbackup/Iil3RwJtYPpGzrPFVjgwUvYmHwTdUk2tExBTvAOEN-tTzXa83LXQIwr8F1oTHM561YQMCpHEqVZWggqE75pjAmduoIoHaBqY7HpnKWbkE6k46QQTeBvOfnFZ0KYLJSIUsXyeS0mCtOMZehMRmBxy_Q8.png)
 
-**Step 2:** Click **Unstake**. **Sign** the transaction using your Web3 wallet.
+**Step 6:** Click **Unstake**. **Sign** the transaction using your Web3 wallet.
 
-**Step 3:** Wait for the escrow period to complete. Observe the Unstaking balance has increased during the escrow period.
+**Step 7:** Wait for the escrow period to complete. Observe the Unstaking balance has increased during the escrow period.
 
 ![](/img/googleusercontentbackup/QZvihKxzjPXs4XBWp8xNZdsc8FmLl7VuJ0m5yom_TqgqiFkpROSVSqEr_4XdMWMmhD-ZcTVOjRYxEaTmwpQzIz87dJdgpNs79jDQvw7TKgNxIXr3P1OJCsWeKZoew4I2iG_6Phy5rlej99HvuoKnY-g.png)
 
-**Step 4:** Observe the Unstaked Beacon Chain balance has increased after the escrow period.
-
-:::warning
-At this point you must exit your beacon chain validators prior to continuing. The validators must have completely exited from the validator queue before continuing. You may use [beaconcha.in](https://beaconcha.in) as a reference during this process.
-:::
-
-After your validators have exited, the Execution Chain amount should increase by the amount of those exited validators.
+**Step 8:** Observe the Unstaked Beacon Chain balance has increased after the escrow period. The Execution Chain amount should have increased by the amount of those exited validators.
 
 The Redeposit button is available at this point to allow the user to Restake back into EigenLayer in case the withdrawal was queued by mistake.
 
-**Step 5:** Once your validator has been exited click Withdraw to complete your deposit. **Sign** the transaction using your Web3 wallet.
+**Step 9:** Once your validator has been exited click Withdraw to complete your deposit. **Sign** the transaction using your Web3 wallet.
```

Comment:
Maybe change the wording, we already told them the validator should be exited before queueing so this is confusing. Maybe "Once your funds clear the escrow". 

Also it should he complete your withdrawal, not deposit right?

Reply:
Good catch! That was a holdover from the old ordering I forgot to fix. Updated now.